src/openrct2/peep/GuestPathfinding.cpp

```diff
@@ -785,7 +786,7 @@ static void peep_pathfind_heuristic_search(
                 if (tileElement->AsPath()->IsWide())
                 {
                     /* Check if staff can ignore this wide flag. */
-                    if (!staff_can_ignore_wide_flag(peep, loc.ToCoordsXYZ(), tileElement))
+                    if (!peep->Is<Staff>() || !static_cast<Staff*>(peep)->CanIgnoreWideFlag(loc.ToCoordsXYZ(), tileElement))
```

Comment:
```suggestion
                    if (!peep->Is<Staff>() || !peep->AsStaff()->CanIgnoreWideFlag(loc.ToCoordsXYZ(), tileElement))
```

Reply:
Deliberately not using `AsStaff` here, per @duncanspumpkin https://github.com/OpenRCT2/OpenRCT2/pull/12907#discussion_r488788491

Reply:
Ah, I didn't realise. Thanks.

Reply:
Yeah but i meant use `As<Staff>`

Reply:
Oh! My bad, you mentioned using `Is` but I didn't spot that there are `As` helpers as well. Will fix.